.github/workflows/ci.yml

```diff
@@ -115,6 +115,8 @@ jobs:
       - run: rustup default ${{ env.RUST_VERSION }}
 
       - uses: Swatinem/rust-cache@v2.2.0
+        with:
+          prefix-key: "2023-02-02-reset-cache"
```

Comment:
hmm... interesting. is this an issue we should report somewhere? cargo maybe? seems to me like an OS update should invalidate the cache automatically, no?

Reply:
FYI I just rebased the branch on top of `master` and removed the cache buster commit since I deleted the relevant caches manually :)